blueprints/component-class/index.js

```diff
@@ -14,7 +14,7 @@ const OCTANE = has('octane');
 // TODO: this should be reading from the @ember/canary-features module
 // need to refactor broccoli/features.js to be able to work more similarly
 // to https://github.com/emberjs/data/pull/6231
-const EMBER_GLIMMER_SET_COMPONENT_TEMPLATE = true;
+const EMBER_GLIMMER_SET_COMPONENT_TEMPLATE = false;
```

Comment:
The flag has been _enabled_ so we want to remove this variable and treat it as if it is always true in places that uses/references it.